diff --git a/node_modules/@react-native-firebase/auth/RNFBAuth.podspec b/node_modules/@react-native-firebase/auth/RNFBAuth.podspec
index a9bf9ec..5352fce 100644
--- a/node_modules/@react-native-firebase/auth/RNFBAuth.podspec
+++ b/node_modules/@react-native-firebase/auth/RNFBAuth.podspec
@@ -1,6 +1,6 @@
 require 'json'
 package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
-appPackage = JSON.parse(File.read(File.join('..', 'app', 'package.json')))
+appPackage = JSON.parse(File.read(File.join('..', 'package.json')))
 
 coreVersionDetected = appPackage['version']
 coreVersionRequired = package['peerDependencies'][appPackage['name']]
